static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -84,17 +91,33 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
                 size={12}
                 title={feedbackItem.project.slug}
               />
-              <TextOverflow>{feedbackItem.project.slug}</TextOverflow>
+              <TextOverflow>{feedbackItem.shortId}</TextOverflow>
+              <DropdownMenu
+                triggerProps={{
+                  'aria-label': t('Short-ID copy actions'),
+                  icon: <IconChevron direction="down" size="xs" />,
+                  size: 'zero',
+                  borderless: true,
+                  showChevron: false,
+                }}
+                position="bottom"
+                size="xs"
+                items={[
+                  {
+                    key: 'copy-url',
+                    label: t('Copy Issue URL'),
+                    onAction: handleCopyUrl,
+                  },
+                  {
+                    key: 'copy-short-id',
+                    label: t('Copy Short-ID'),
+                    onAction: handleCopyShortId,
+                  },
+                ]}
+              />
             </Flex>
           </Flex>
           <Flex gap={space(1)} align="center" wrap="wrap">
-            <Button
-              title={t('Copy link to this feedback')}
-              size="xs"
-              onClick={copyLink}
-              aria-label={t('Copy Link')}
-              icon={<IconLink />}
-            />
```

Comment:
I removed the stand-alone button and put it inside the dropdown, it's copy+pasted from Issues

I think we should still re-vamp the details header here, which will be in a followup.